package.json

```diff
@@ -35,67 +35,68 @@
     "changelog": "standard-changelog"
   },
   "dependencies": {
-    "@angular/animations": "4.1.3",
-    "@angular/common": "4.1.3",
-    "@angular/compiler": "4.1.3",
-    "@angular/core": "4.1.3",
-    "@angular/forms": "4.1.3",
-    "@angular/http": "4.1.3",
-    "@angular/platform-browser": "4.1.3",
-    "@angular/platform-browser-dynamic": "4.1.3",
-    "@angular/platform-server": "4.1.3",
-    "@angular/router": "4.1.3",
-    "core-js": "2.4.1",
-    "highlight.js": "9.10.0",
-    "lodash": "4.17.4",
-    "ng2-completer": "1.2.2",
-    "normalize.css": "5.0.0",
-    "rxjs": "5.4.0",
-    "zone.js": "0.8.10"
+    "@angular/animations": "6.0.2",
+    "@angular/common": "6.0.2",
+    "@angular/compiler": "6.0.2",
+    "@angular/core": "6.0.2",
+    "@angular/forms": "6.0.2",
+    "@angular/http": "6.0.2",
+    "@angular/platform-browser": "6.0.2",
+    "@angular/platform-browser-dynamic": "6.0.2",
+    "@angular/platform-server": "6.0.2",
+    "@angular/router": "6.0.2",
+    "core-js": "2.5.6",
+    "highlight.js": "9.12.0",
+    "lodash": "4.17.10",
+    "ng2-completer": "1.6.3",
+    "normalize.css": "8.0.0",
+    "rxjs": "6.1.0",
+    "rxjs-compat": "^6.1.0",
```

Comment:
yes we need it because "ng2-completer" depends on rxjs 5.1 or higher so it will also reference to rxjs 6 with the breaking changes, so to keep it work we need it :)

Reply:
got it